js/runner.go

```diff
@@ -363,57 +362,51 @@ type VU struct {
 	TLSConfig *tls.Config
 	ID        int64
 	Iteration int64
+	RunMutex  *sync.Mutex
```

Comment:
I would prefer if this is unexported, there is no need for it to be either way

Reply:
Resolved by 1c2e79ce.